### PR TITLE
Miscellaneous improvement/cleanup in el args

### PR DESCRIPTION
**Pull request by g11tech**
- enable snap sync for nethermind
- cleanup eth endpoints in geth and nethermind and erigon (erigon yet to be tested)
- cleaned up kiln.vars
- removed merge ttd override from the merged networks as no more relevant to them, still retained in mainnet as there could be last minute requirement

@philknows can you also test geth/nethermind for mainnet and confirm this is working and no issues

also would be thankful if you can see if erigon also works, didn't test it yet.

### CODE REVIEW

File: fixed.vars [hunk 1]
```diff
@@ -5,17 +5,17 @@ LODESTAR_FIXED_VARS="--dataDir /data/lodestar --execution.urls http://127.0.0.1:
...
+NETHERMIND_FIXED_VARS="--datadir /data/nethermind --Sync.SnapSync=true --Merge.Enabled=true --Init.DiagnosticMode=None --JsonRpc.Enabled=true --JsonRpc.Host=0.0.0.0 --JsonRpc.AdditionalRpcUrls \"http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client\" --JsonRpc.JwtSecretFile /data/jwtsecret"
 
 # Data dir will be mounted in /data/geth
-GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --http.api engine,net,eth,web3 --http.port 8545 --allow-insecure-unlock --http.addr 0.0.0.0 --http.corsdomain \"*\" --http.vhosts \"*\" --authrpc.port=8551"
+GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --authrpc.addr 0.0.0.0 --authrpc.corsdomain \"*\" --authrpc.vhosts \"*\" --authrpc.addr 0.0.0.0 --authrpc.vhosts \"*\" --authrpc.port=8551"
```
Comment (philknows) [inline fixed.vars:11]: Geth is complaining about the flag `--authrpc.corsdomain \"*\"`. I've looked in the docs and could only find `--http.corsdomain \"*\"`. This fixed it, but unsure if the need for it on authrpc port is needed/or exists?

Comment (philknows) [reply fixed.vars:11]: Used the docs here: https://geth.ethereum.org/docs/interface/command-line-options
It's updated only to v1.10.19 and not the current v1.10.23 though FYI

File: fixed.vars [hunk 2]
```diff
@@ -5,17 +5,17 @@ LODESTAR_FIXED_VARS="--dataDir /data/lodestar --execution.urls http://127.0.0.1:
...
 
 # Data dir will be mounted in /data/besu
 BESU_FIXED_VARS="--data-path=/data/besu --engine-jwt-secret=/data/jwtsecret --rpc-http-enabled=true --rpc-http-api=ADMIN,CLIQUE,MINER,ETH,NET,DEBUG,TXPOOL,TRACE --rpc-http-host=0.0.0.0 --rpc-http-port=8545 --engine-rpc-port=8551 --rpc-http-cors-origins=\"*\" --host-allowlist=\"*\" --engine-host-allowlist=\"*\" --p2p-enabled=true --engine-jwt-enabled=true"
 
 # Data dir will be mounted in /data/erigon, private api addr has been switched off because it collides
 # with lodestar prometheus endpoint. If required make adjustments accordingly
-ERIGON_FIXED_VARS="--datadir=/data/erigon --private.api.addr \"\" --http --http.api \"admin,engine,net,eth\" --http.port=8545 --http.addr 0.0.0.0 --http.corsdomain \"*\" --ws --engine.port=8551 --engine.addr=0.0.0.0 --authrpc.jwtsecret=/data/jwtsecret --allow-insecure-unlock --batchSize=32m"
\ No newline at end of file
+ERIGON_FIXED_VARS="--datadir=/data/erigon --private.api.addr \"\" --http --engine.port=8551 --engine.addr=0.0.0.0 --authrpc.jwtsecret=/data/jwtsecret --batchSize=32m"
```
Comment (philknows) [inline fixed.vars:21]: `--engine.port=8551` and `--engine.addr=0.0.0.0` is now `--authrpc.port=8551` and `--authrpc.addr=0.0.0.0`

File: fixed.vars (hunk 1 above)
Comment (g11tech) [reply fixed.vars:11]: let me see because i am running geth with this locally

Comment (g11tech) [reply fixed.vars:11]: yes you are correct! its http.cors domain!

**Pull request merged**